en/drafts/originals/space-place-gazetteers.md

```diff
@@ -25,122 +25,111 @@ doi: XX.XXXXX/phen0000
+- Identify scenarios for which creating a gazetteer may be preferable to using a geographic information system   
+- Transform a historical text into a gazetteer   
+- Share a gazetteer with other platforms to enhance it and use it for analytical purposes   
 
-No coding experience is needed to complete this lesson.  You should be comfortable designing and using spreadsheets and you will need access to a spreadsheet platform such as Microsoft Excel, Google Sheets, or LibreOffice Calc.
+### Prerequisites
 
-## Historical Example
+No coding experience is needed to complete this lesson. You should simply be comfortable designing and using spreadsheets, and you will need access to a spreadsheet platform such as Microsoft Excel, Google Sheets, or LibreOffice Calc.
 
-This lesson will show you how to create a gazetteer based on the online [Itinerary of Benjamin of Tudela](https://depts.washington.edu/silkroad/texts/tudela.html), an English translation of a Hebrew-language itinerary composed by Benjamin of Tudela (1130-1173), a Jewish traveler who journeyed between the Iberian Peninsula and West Asia in the twelfth century. Benjamin transited through three hundred cities along his route, recording information about geography, ethnography, commerce, Jewish life, and Jewish-Muslim relations.[^2] This text is a major work of medieval geography and Jewish history. You will build a gazetteer of places that Benjamin visited. 
+### Example Case Study
 
-This example will teach you to extract place names from written historical texts and use them to build a succinct gazetteer. The waypoints along Benjamin’s journey are cities with synagogues, so the lesson will explain how to build a gazetteer that includes historic place names as well as other feature types that are important in the historical record. This fulfills two lesson components, demonstrating firstly why and how a scholar might choose to build a basic gazetteer, and secondly how a gazetteer can support historical analysis.
+This lesson will show you how to create a gazetteer based on the online [_Itinerary of Benjamin of Tudela_](https://depts.washington.edu/silkroad/texts/tudela.html), an English translation of a Hebrew-language itinerary composed by Benjamin of Tudela (1130-1173), a Jewish traveler who journeyed between the Iberian Peninsula and West Asia in the 12th century. Benjamin transited through 300 cities along his route, recording information about geography, ethnography, commerce, Jewish life, and Jewish-Muslim relations.[^3] This text is a major work of medieval geography and Jewish history. This lesson will teach you to extract place names from this written historical text and use them to build a succinct gazetteer of the places that Benjamin visited, including their historic names and other feature types which are essential to the historical record. 
 
-## Background: Space, Place, Gazetteers, and Knowledge Organization Systems
+## Initial Considerations
```

Comment:
It was especially important to reword this heading, since I removed the mention of KOS from this section. I thought 'Initial Considerations' made sense, because each sub-section is a discussion of concepts or formats to think about (concept of place, whether to build a gazetteer or GIS, whether to group names or not, and why you'd want to use the LP/LP-TSV formats).

Reply:
Good changes.